Joy sql normalization (#269)
arcus/education_modules: sql_normalization/sql_normalization.md

<!--
author:   Joy Payton
email:    paytonk@chop.edu
version:  1.0.0
module_template_version: 3.0.0
language: en
narrator: US English Female

> title: SQL Normalization

Reviewer: ```suggestion
title: Database Normalization
```

> comment: Learn about the concept of normalization and why it's important for SQL.

Reviewer: Bigger question here... yes this is how SQL is organized, but it is also just pure information theory.

I can see this module being assigned after the "tidy data" module, and it really is more about data management than about learning SQL
```suggestion
comment: Learn about the concept of normalization and why it's important for organizing complicated data.
```
```suggestion
comment: Learn about the concept of normalization and why it's important for SQL.
```

Author: ooooh you are so right

> long_description: Usually, data in a SQL database is organized into multiple interrelated tables with as little data repetition as possible. This module teaches underlying data considerations and explains how data is organized in SQL by introducing the concepts of one-to-many data relationships and normalization.

Reviewer: ```suggestion
long_description: Usually, data in a relational database like SQL is organized into multiple interrelated tables with as little data repetition as possible. This module teaches underlying data considerations and explains how data is organized in SQL by introducing the concepts of one-to-many data relationships and normalization.
```

estimated_time: 40 minutes

@learning_objectives  
After completion of this module, learners will be able to:

- Explain the significance of "one to many" data relationships and how these relationships affect data organization
- Describe how a normalized database is typically organized
- Explain how data can be linked between tables and define "primary keys" and "foreign keys"

@end

link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css

script: https://kit.fontawesome.com/83b2343bd4.js
-->

> # SQL Normalization

Reviewer: ```suggestion
# Database Normalization
```

<div class = "overview">

## Overview
@comment

**Is this module right for me?** @long_description

**Estimated time to completion:** @estimated_time

> **Pre-requisites**  Learners should have experience writing SQL code on single tables.  If you have successfully used a "SELECT... FROM... WHERE" SQL statement on a single table, this is sufficient for this module. If you need to develop basic SQL fluency we recommend our module [SQL Basics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/sql_basics/sql_basics.md).  

Reviewer: ```suggestion
**Pre-requisites**  Learners should be familiar with tabular data and have an understanding of what makes data "tidy." Although SQL will be referenced throughout this module, no knowledge of SQL, other than the fact that it is a way to store large databases, is required. 
```

> **Pre-requisites**  Learners should have experience writing SQL code on single tables.  If you have successfully used a "SELECT... FROM... WHERE" SQL statement on a single table, this is sufficient for this module. If you need to develop basic SQL fluency we recommend our module [SQL Basics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/sql_basics/sql_basics.md).  

Author: ```suggestion
**Pre-requisites**  Learners should have experience working with data in tables.  This could included working with .csv files, SQL databases, R data frames, REDCap instruments, or other ways that data can be collected in tables. 
```
